test/models/user_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name: "Example User", email: "user@example.com",
+                     password: "foobar", password_confirmation: "foobar")
```

Comment:
I think our style is to do:

```suggestion
user = User.new(
  name: "Example User",
  email: "user@example.com",
  password: "foobar",
  password_confirmation: "foobar"
)
```